File: src/main/java/dev/sky_lock/mocar/command/HelpCommand.java
package dev.sky_lock.mocar.command;

import dev.sky_lock.mocar.MoCar;
import dev.sky_lock.mocar.Permission;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

/**
 * @author sky_lock
 */

public class HelpCommand implements ICommand {

    @Override
    public void execute(CommandSender sender, Command cmd, String[] args) {
        sender.sendMessage(MoCar.PREFIX + "-----------Help-------------");
        sender.sendMessage(MoCar.PREFIX + "/mocar - 利用可能なコマンドを表示します");
        if (Permission.ADMIN_COMMAND.obtained(sender)) {
            sender.sendMessage(MoCar.PREFIX + "/mocar give [target] [carId] - 指定した車種の車を指定したプレイヤーの位置にスポーンさせます");
            sender.sendMessage(MoCar.PREFIX + "/mocar search [target] - 所有している車の現在地を表示します");
            sender.sendMessage(MoCar.PREFIX + "/mocar toaway [target] - 指定したプレイヤーの車をアイテム化します");
            sender.sendMessage(MoCar.PREFIX + "/mocar edit - カーモデルエディタを開きます");
            sender.sendMessage(MoCar.PREFIX + "/mocar removemodel(rm) [carId] - 指定したidの車種を削除します");
            sender.sendMessage(MoCar.PREFIX + "/mocar listmodel(lm) - 全ての車種を表示します");
            sender.sendMessage(MoCar.PREFIX + "/mocar reload - CarModelコンフィグから車種情報を読み込みます");
            sender.sendMessage(MoCar.PREFIX + "/mocar debug - デバッグ用コマンド");
            sender.sendMessage(MoCar.PREFIX + "----------------------------");
            return;
        }
        sender.sendMessage(MoCar.PREFIX + "/mocar search - 所有している車の現在地を表示します");
        sender.sendMessage(MoCar.PREFIX + "/mocar towaway - 所有している車をアイテム化します");
        sender.sendMessage(MoCar.PREFIX + "----------------------------");
    }
}

File: src/main/kotlin/dev/sky_lock/pocketlifevehicle/command/HelpCommand.kt
package dev.sky_lock.pocketlifevehicle.command

import dev.sky_lock.pocketlifevehicle.Permission
import dev.sky_lock.pocketlifevehicle.extension.chat.sendVehiclePrefixedRawMessage
import org.bukkit.command.Command
import org.bukkit.command.CommandSender

/**
 * @author sky_lock
 */
class HelpCommand : ICommand {
    override fun execute(sender: CommandSender, cmd: Command, args: Array<String>) {
        sender.sendVehiclePrefixedRawMessage("-----------コマンドヘルプ-------------")
        sender.sendVehiclePrefixedRawMessage("/vehicle - 利用可能なコマンドを表示します")
        if (Permission.ADMIN_COMMAND.obtained(sender)) {
            sender.sendVehiclePrefixedRawMessage("/vehicle world (add|remove|list) - ワールドで乗り物が使用できるかどうかを設定します")
            sender.sendVehiclePrefixedRawMessage("/vehicle give [modelId] [target] - モデルアイテムをプレイヤーにインベントリーに追加します")
            sender.sendVehiclePrefixedRawMessage("/vehicle spawn [modelId] [target] - モデルをプレイヤーの位置にスポーンさせます")
            sender.sendVehiclePrefixedRawMessage("/vehicle search [target] - 乗り物の現在地を表示します")
            sender.sendVehiclePrefixedRawMessage("/vehicle forgive [target] - プレイヤーの駐車違反登録を解除します")
            sender.sendVehiclePrefixedRawMessage("/vehicle pop [target] - プレイヤーの乗り物をアイテム化します")
            sender.sendVehiclePrefixedRawMessage("/vehicle model - モデルエディタを開きます")
            sender.sendVehiclePrefixedRawMessage("/vehicle event (clear|unlock|lock) - イベント車両用コマンドです")
            sender.sendVehiclePrefixedRawMessage("/vehicle reload - ファイルからモデル情報を読み書きします")
            sender.sendVehiclePrefixedRawMessage("/vehicle debug - デバッグ用コマンド")
            sender.sendVehiclePrefixedRawMessage("-----------------------------------")
            return
        }
        sender.sendVehiclePrefixedRawMessage("/vehicle search - モデルの現在地を表示します")
        sender.sendVehiclePrefixedRawMessage("/vehicle pop - 乗り物をアイテム化します")
        sender.sendVehiclePrefixedRawMessage("-----------------------------------")

    }
}